sketch-core/src/DataHelpers.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright 2015-present Ampersand Technologies, Inc.
+ */
+
+import * as TableDef from './TableDef';
+import * as Types from './types';
+
+import * as ObjUtils from 'amper-utils/dist/objUtils';
+import { Stash, StashOf } from 'amper-utils/dist/types';
```

Comment:
FYI StashOf no longer exists, I switched it to just Stash in amper-utils.

Reply:
yeah,  I removed a bunch so far, but I guess I missed this one.